README.md

```diff
@@ -1 +1,126 @@
-# book-store
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 Math Magicians <a name="about-project"></a>
+<br>
```

Comment:
- [ ] *You have done a great job so far 👏* however, it would be better to change the project title in your readme to make it more professional 🙏 
<img width="965" alt="Screen Shot 2023-09-23 at 18 57 34" src="https://github.com/MohammadYaser/book-store/assets/102757126/0ae40d4c-2715-4284-be4e-826f7ae59874">